.github/workflows/update-sdk-version.yml

```diff
@@ -68,8 +68,8 @@ jobs:
           ./updateSdkVersions.sh all
           BRANCH=bump-sdk-versions-${SDK_VERSION}
           git checkout -b ${BRANCH}
-          git config user.name "Akka Bot"
-          git config user.email "github-actions@users.noreply.github.com"
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
Hm, but how did that work before, where does this email come from (are `[]` even allowed in email addresses)?

Reply:
> Hm, but how did that work before

From what I can see in the GH actions this part of the workflow was never used, in this repo.
This comes from `akka-jdk` repo but there I can't see `Actions`. Maybe because is archived. 
Not sure it worked in `akka-jdk` before. I vaguely remember it did, but far from sure. 

> where does this email come from (are [] even allowed in email addresses)?

From what I gathered
https://github.com/actions/checkout/pull/1184/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5
https://github.com/actions/checkout/blob/main/README.md#push-a-commit-using-the-built-in-token

Reply:
Huh, strange that it is the same for anyone using GitHub actions and not something project specific.